Source/WebGPU/WebGPU/Device.mm

```diff
@@ -239,14 +239,18 @@ static void captureFrame(id<MTLDevice> captureObject)
     return eye == WGPUXREye_Right ? m_xrSubImages[1] : m_xrSubImages[0];
 }
 
+void Device::makeInvalid()
+{
+    m_device = nil;
+    m_defaultQueue->makeInvalid();
+}
+
 void Device::loseTheDevice(WGPUDeviceLostReason reason)
 {
     m_device = nil;
 
     m_adapter->makeInvalid();
 
-    makeInvalid();
```

Comment:
this call was redundant since previously makeInvalid just set m_device = nil which occurs 4 lines above